dtls/server.go

```diff
@@ -36,7 +37,7 @@ type GoPoolFunc = func(func()) error
 
 type BlockwiseFactoryFunc = func(getSendedRequest func(token message.Token) (blockwise.Message, bool)) *blockwise.BlockWise
 
-type OnNewClientConnFunc = func(cc *client.ClientConn)
+type OnNewClientConnFunc = func(cc *client.ClientConn, dtlsConn *dtls.Conn)
```

Comment:
Pls add comment that `dtlsConn *dtls.Conn` below to `client.ClientConn` and `dtlsConn.Close()` is forbidden -> cause panic for duplicity close.